src/core_plugins/kibana/public/visualize/editor/_editor.scss

```diff
@@ -0,0 +1,46 @@
+.visEditor {
+  @include flex-parent();
+
+  @include euiBreakpoint('l', 'xl') {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+  }
+
+  @include euiBreakpoint('xs', 's', 'm') {
+    .visualization {
+      // While we are on a small screen the visualization is below the
+      // editor. In this cases it needs a minimum height, since it would otherwise
+      // maybe end up with 0 height since it just gets the flexbox rest of the screen.
+      min-height: 400px;
+    }
+  }
+
+  // SASSTODO: Is this even an element anymore, or should it be .visualize?
+  /* 1. Without setting this to 0 you will run into a bug where the filter bar modal is hidden under
+a tilemap in an iframe: https://github.com/elastic/kibana/issues/16457 */
+  > visualize {
```

Comment:
As you mentioned above, this should actually be `.visualize` and got broken in some of the Angular refactoring. I will already fix this on master so we can pull the fix in 6.4.[1|2].